content/developer/reference/backend/orm_changelog.rst

```diff
@@ -0,0 +1,15 @@
+
+.. _reference/orm_changelog:
+
+=============
+ORM Changelog
+=============
+
+
+version 15.2
+============
+
+specific index types on fields
+------------------------------
+With https://github.com/odoo/odoo/pull/83274 and https://github.com/odoo/odoo/pull/83015 developers can now define what type of indexes can be used by postgresql on fields
+:ref:`see the index property of Field <reference/fields>`.
```

Comment:
Assuming this is in large parts for external consumption, it should probably have 15.1 as well?

Reply:
No, I mean the changelog for saas-15.1 should be included.